homeassistant/components/icloud/__init__.py

```diff
@@ -69,48 +63,6 @@
     }
 )
 
-ACCOUNT_SCHEMA = vol.Schema(
-    {
-        vol.Required(CONF_USERNAME): cv.string,
-        vol.Required(CONF_PASSWORD): cv.string,
-        vol.Optional(CONF_WITH_FAMILY, default=DEFAULT_WITH_FAMILY): cv.boolean,
-        vol.Optional(CONF_MAX_INTERVAL, default=DEFAULT_MAX_INTERVAL): cv.positive_int,
-        vol.Optional(
-            CONF_GPS_ACCURACY_THRESHOLD, default=DEFAULT_GPS_ACCURACY_THRESHOLD
-        ): cv.positive_int,
-    }
-)
-
-CONFIG_SCHEMA = vol.Schema(
```

Comment:
We keep this around and use:

```py
CONFIG_SCHEMA: = cv.removed(DOMAIN, raise_if_present=False)
```

It causes issues if it is configured, but the integration schema doesn't exists (triggers safe mode).